app/views/devise/mailer/unlock_instructions.html.erb

```diff
@@ -1,7 +1,8 @@
-<p>Hello <%= @resource.email %>!</p>
+<% require 'devise/version' %>
+<p><%= t('.greeting', recipient: @resource.email) %></p>
 
-<p>Your account has been locked due to an excessive amount of unsuccessful sign in attempts.</p>
+<p><%= t('.message') %></p>
 
-<p>Click the link below to unlock your account:</p>
+<p><%= t('.instruction') %></p>
 
-<p><%= link_to 'Unlock my account', unlock_url(@resource, unlock_token: @token) %></p>
+<p><%= link_to t('.action'), unlock_url(@resource, unlock_token: (Devise::VERSION.start_with?('3.') ? @token :@resource.unlock_token)) %></p>
```

Comment:
this is from devise's own generator, I just copied it from the new generated templates, just in case